en/setup/dev_env_windows_msys.md

```diff
@@ -1,8 +1,10 @@
 # Windows Msys Toolchain
 
+> **Note** This solution is currently broken.
```

Comment:
@MaEtUgR It would be good for each solution to include the high level info in the introduction. In this case I've just said broken, but actually it would be good to say something about the solutions. I know nothing about Msys, but basically "supports PX4 nuttx builds, jMAVSim". Installer creates non-portable toolchain and includes start menu items .... whatever, basically you see my point that it is worth having "method evaluation info" up the top.